File: java/semantickernel-api/src/main/java/com/microsoft/semantickernel/templateengine/blocks/BlockTypes.java
// Copyright (c) Microsoft. All rights reserved.
package com.microsoft.semantickernel.templateengine.blocks;

public enum BlockTypes {
    /** Undefined block type */
    Undefined(0),
    /** Text block type */
    Text(1),
    /** Code block type */
    Code(2),
    /** Variable block type */
    Variable(3),
    /** Value block type */
    Value(4),
    /** Function block type */
    FunctionId(5);

    BlockTypes(int i) {}
}

File: dotnet/src/Extensions/TemplateEngine.PromptTemplateEngine/Blocks/BlockTypes.cs
// Copyright (c) Microsoft. All rights reserved.

namespace Microsoft.SemanticKernel.TemplateEngine.Prompt.Blocks;

internal enum BlockTypes
{
    Undefined = 0,
    Text = 1,
    Code = 2,
    Variable = 3,
    Value = 4,
    FunctionId = 5,
    NamedArg = 6,
}
